classes/task/emit_task.php

```diff
@@ -50,10 +50,13 @@ public function execute() {
         $store = new store($manager);
 
         $events = $DB->get_records('logstore_xapi_log');
-        $store->process_events($events);
-
-        $DB->delete_records_list('logstore_xapi_log', 'id', array_keys($events));
-
+        $store_return = $store->process_events($events);
+		foreach(array_keys($store_return) as $event_id)
+			{
+			if ($store_return[$event_id] == 'success')
```

Comment:
`\mod_quiz\event\attempt_reviewed` as shown in the [Translator Controller](https://github.com/LearningLocker/Moodle-xAPI-Translator/blob/master/src/Controller.php#L33)